### ISSUE TITLE
Add option for ssh plugin to use different node attribute for ssh address

### ISSUE THREAD

**Issue by atheiman** [rundeck/rundeck#2158]
according to the ssh plugin docs http://rundeck.org/docs/plugins-user-guide/ssh-plugins.html there is no way to tell the ssh plugin to use a different node attribute other than `hostname` for remote logins. It would be nice if this could be configurable to whatever the admin wants, set in the project config file. For example, to use the `fqdn` attribute:

```INI
project.ssh-address-attribute=fqdn
```

**Comment by gschueler** [comment]
is this to integrate with your resource model source better?

**Comment by atheiman** [comment]
Yes, to use a different attribute that's returned on the node. Unfortunately in some environments, ssh address needs to be ipaddress or fqdn.

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
